initfiles/systemd/gitautodeploy.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=GitAutoDeploy
+
+[Service]
+User=root
```

Comment:
Hard to say... I guess `www-data` should be fine, as long as it exists, so make sure to add that as a node in the `README.md` too!